server/statistics/hot_peer_cache.go

```diff
@@ -116,13 +116,17 @@ func (f *hotPeerCache) CheckRegionFlow(region *core.RegionInfo, storesStats *Sto
 	byteRate := totalBytes / float64(interval)
 	keyRate := totalKeys / float64(interval)
 
-	var tmpItem *HotPeerStat
+	var cacheItem *HotPeerStat
+	for _, storeID := range storeIDs {
+		cacheItem = f.getOldHotPeerStat(region.GetID(), storeID)
```

Comment:
Is it different from the original logic?

Reply:
I think we only need to get a cached item whether expired. cc @nolouch